flycheck.el

```diff
@@ -8380,6 +8380,9 @@ See URL `http://jruby.org/'."
   :modes (enh-ruby-mode ruby-mode)
   :next-checkers ((warning . ruby-rubylint)))
 
+(flycheck-def-args-var flycheck-cargo-rustc-args (rust-cargo)
+  :package-version '(flycheck . "0.24"))
```

Comment:
This is supposed to indicate the flycheck version number that *introduced* this variable.  In this case, this should be "30" rather than "0.24".